tests/test_modeling_gpt_neo.py

```diff
@@ -0,0 +1,511 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch GPT Neo model. """
+
+
+import unittest
+
+from transformers import is_torch_available
+from transformers.testing_utils import require_torch, slow, torch_device
+
+from .test_configuration_common import ConfigTester
+from .test_generation_utils import GenerationTesterMixin
+from .test_modeling_common import ModelTesterMixin, floats_tensor, ids_tensor, random_attention_mask
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        GPT_NEO_PRETRAINED_MODEL_ARCHIVE_LIST,
+        GPT2Tokenizer,
+        GPTNeoConfig,
+        GPTNeoForCausalLM,
+        GPTNeoModel,
+    )
+
+
+class GPTNeoModelTester:
```

Comment:
It would be great if we could also test the functions:
`create_attention_mask`, `look_around`, `shift`

a bit more in-detail